transports/rabbitmq/delayed-delivery.md

```diff
@@ -21,58 +21,111 @@ N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.destination
 
 Where `N` is either `0` or `1`, representing the delay value in binary, and `destination` is the name of endpoint the delayed message will be sent to.
 
+As an example, a delay of 10 seconds (`1010` in binary) on a message bound for the `destination` queue would be encoded with a routing key of:
+
+```
+0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.1.0.1.0.destination
+```
+
 
 ### Delay levels
 
 Each exchange/queue pair that makes up a level represents one bit of the total delay value. By having 28 of these levels, corresponding to `2^27` through `2^0`, the delay infrastructure can delay a message for any value that can be represented by a 28-bit number. With 28 total levels, the maximum delay value is 268,435,455 seconds, or about 8.5 years.
 
 A delay level is created by declaring a topic exchange that is bound to a queue with a routing key of `1`, and to the exchange corresponding to `level - 1` with a routing key of `0`. The queue for the delay level is declared with an [`x-message-ttl`](https://www.rabbitmq.com/ttl.html) value corresponding to `2^level` seconds. The queue is also declared with an [`x-dead-letter-exchange`](https://www.rabbitmq.com/dlx.html) value corresponding to the `level - 1` exchange, so that when a message in the queue expires, it will be routed to the `level - 1` exchange.
 
+The delay levels are connected in this manner, from highest (27) to lowest (0). Each delay level's routing key's add wildcards as needed so that they are looking at the portion of the message's routing key that corresponds to its delay level.
+
+The following diagram illustrates the relationships between exchanges and queues, which are connected by bindings and `x-message-ttl` delay values. It's important to note the wildcard rules for RabbitMQ binding expressions, where *word* describes a dot-delimited segment:
+
+* `*` substitutes for exactly one word
+* `#` substitutes for 0 or more words
+
+<style>
```

Comment:
What's this for?

Reply:
The style is to get the bindings to use a fixed-width font because otherwise the repeated star dot star dot star looks like garbage...

![image](https://user-images.githubusercontent.com/427110/44233115-98618a00-a168-11e8-9b21-e6526fe98a1f.png)

(Image because GitHub will make bold periods otherwise)

Reply:
I guess it's ok to reference Consolas?